### PR TITLE
Auth todos pr0

### PR DESCRIPTION
**Pull request by RainingComputers**

### CODE REVIEW

**Comment by alexsanjoseph** [comment]
Can you ignore .DS_store instead? (it's a osx artifact)

**Comment by alexsanjoseph** [comment]
(you might have to do a https://stackoverflow.com/questions/7527982/applying-gitignore-to-committed-files)

File: meiki_server/auth/auth_test.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ func (s *AuthTestSuite) clean() {
 func (s *AuthTestSuite) SetupTest() {
 	log.Initialize()
 
-	s.ctx, s.cancel = context.WithTimeout(context.Background(), 500*time.Millisecond) // 500ms might not be enough on some systems?
+	s.ctx, s.cancel = context.WithTimeout(context.Background(), 500000000000*time.Millisecond) // 500ms might not be enough on some systems?
```
Comment (alexsanjoseph) [inline meiki_server/auth/auth_test.go]: does this prevent it from timing out in situations where the test services are not initialized?

File: meiki_server/auth/routes_test.go [hunk 2]
```diff
@@ -113,7 +113,7 @@ func (s *AuthRoutesTestSuite) TestRoutesScenario() {
 		Password: "no-shnoo",
 	})
 	req, _ = http.NewRequest("POST", "/login", bytes.NewBuffer(badCredentialsBody2))
-	s.assertStatusCode(req, 401)
+	s.assertStatusCode(req, 400)
```
Comment (alexsanjoseph) [inline meiki_server/auth/routes_test.go]: Why is this a 400 and not 401?

**Comment by alexsanjoseph** [comment]
LGTM except the nitpicks

File: meiki_server/auth/auth_test.go (hunk 1 above)
Comment (RainingComputers) [reply meiki_server/auth/auth_test.go]: Yeah, will change back

File: meiki_server/auth/routes_test.go (hunk 2 above)
Comment (RainingComputers) [reply meiki_server/auth/routes_test.go]: Yeah, will change it back to 401

File: meiki_server/auth/auth_test.go (hunk 1 above)
Comment (RainingComputers) [reply meiki_server/auth/auth_test.go:35]: Addressed in next PR

File: meiki_server/auth/routes_test.go (hunk 2 above)
Comment (RainingComputers) [reply meiki_server/auth/routes_test.go]: Addressed in next PR

**Comment by alexsanjoseph** [comment]
LGTM

**Pull request merged**